proposal.md

```diff
@@ -21,7 +21,6 @@ Our objective is to determine whether or not these methods can be trained with s
 * ADAGE/eADAGE
 * GANs + clinical data
 * Variational Autoencoders
-* Scleroderma pubs -> cell lineage importance
```

Comment:
will have a separate commit with this content. CA paper is in `metadata.md` - going to try to discuss those topics here (except maybe deep-review, which may go to the sharing section) depending on word count